lua/trunk.lua

```diff
@@ -265,13 +320,27 @@ local function start()
 					return
 				end
 
-				local bufname = vim.fs.basename(vim.api.nvim_buf_get_name(0))
 				local handle = io.popen("command -v timeout")
 				local timeoutResult = handle:read("*a")
 				handle:close()
 				-- Stores current buffer in a temporary file in case trunk fmt fails so we don't overwrite the original buffer with an error message.
 				local tmpFile = os.tmpname()
 				local tmpFormattedFile = os.tmpname()
+				local trunkFormatCmd = table.concat(executionTrunkPath(), " ") .. " format-stdin %:p"
+				if checkCliVersion("1.17.2") then
```

Comment:
Does this mean we're running `trunk version` every time a new buffer is open? I would much prefer to have a global variable for this. I want to avoid shelling out as much as possible